newIDE/app/src/Utils/UpdateDiscordRichPresence.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import optionalRequire from './OptionalRequire';
+const electron = optionalRequire('electron');
+const ipc = electron ? electron.ipcRenderer : null;
+
+/*::
+type Params = {
+  project?: gdProject,
+  resetDate?: boolean,
+};
+*/
+
+const updatePresence = (params /*: ?Params */) => {
```

Comment:
Use the same name as what you would be using outside the module, to allow this function to be searched in the whole codebase. Otherwise, someone would search for `updateDiscordRichPresence` and would never find it (sure, it could use an IDE like VSCode and Ctrl+click on it... but a codebase should be searchable without an IDE).